### PR TITLE
Add location data to ghost verb logs

### PR DESCRIPTION
**Pull request by Wayland-Smithy**
## About The Pull Request

Title.

## Why It's Good For The Game

Admins can check logs to see where ghosting happened.

## Changelog
:cl:
admin: Game logs now save the location of mob ghostings.
/:cl:

### CODE REVIEW

**Comment by BraveMole** [comment]
totally not because of me

**Review by SplinterGP — APPROVED** [review]

File: code/modules/mob/living/living_verbs.dm [hunk 1]
```diff
@@ -65,7 +65,7 @@
 		return
 
 	set_resting(TRUE)
-	log_game("[key_name(usr)] has ghosted.")
+	log_game("[key_name(usr)] has ghosted at [AREACOORD(usr)].")
```
Comment (SplinterGP) [inline code/modules/mob/living/living_verbs.dm:68]: even better 
```suggestion
	log_game("[key_name(usr)] has ghosted at [AREACOORD(usr)] as [src].")
```

**Review by SplinterGP — CHANGES_REQUESTED** [review]

**Comment by SplinterGP** [comment]
my github bonked hard

File: code/modules/mob/living/living_verbs.dm (hunk 1 above)
Comment (SplinterGP) [inline code/modules/mob/living/living_verbs.dm:68]:
```suggestion
	log_game("[key_name(usr)] has ghosted at [AREACOORD(usr)] as [src].")
```

**Review by SplinterGP — CHANGES_REQUESTED** [review]

**Review by SplinterGP — APPROVED** [review]

File: code/modules/mob/living/living_verbs.dm [hunk 2]
```diff
@@ -65,7 +65,7 @@
 		return
 
 	set_resting(TRUE)
-	log_game("[key_name(usr)] has ghosted.")
+	log_game("[key_name(usr)] has ghosted at [AREACOORD(usr)] as [src].")
```
Comment (TiviPlus) [inline code/modules/mob/living/living_verbs.dm:68]:
```suggestion
	log_game("[key_name(usr)] has ghosted at [AREACOORD(usr)]")
```
key name already does this

Comment (Wayland-Smithy) [reply code/modules/mob/living/living_verbs.dm:68]: maint code part 2 the undoing

File: code/modules/mob/living/living_verbs.dm [hunk 3]
```diff
@@ -65,7 +65,7 @@
 		return
 
 	set_resting(TRUE)
-	log_game("[key_name(usr)] has ghosted.")
+	log_game("[key_name(usr)] has ghosted at [AREACOORD(usr)]")
```
Comment (Wayland-Smithy) [inline code/modules/mob/living/living_verbs.dm:68]:
```suggestion
	log_game("[key_name(usr)] has ghosted at [AREACOORD(usr)].")
```

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**